client/src/components/settings/toggle-setting.css

```diff
@@ -78,52 +82,53 @@
   cursor: default;
 }
 
-.toggle-radio-group {
-  display: flex;
-  align-items: start;
-}
-
 .toggle-radio-group label {
-  font-weight: normal;
-  display: flex;
-  align-items: center;
+  margin: 0px 8px 8px 0;
```

Comment:
The bottom margin doesn't seem to be making any difference here, so I think we can get rid of it completely. Do we really need a right margin on both `label`s? If the intent is to put more space between the two `label`s then I think we should use a selector that specifies that. Something like `.toggle-radio-group label + label` and then use `margin-inline-start` to add the spacing.

Reply:
Does this look good?
![Screenshot 2023-04-23 213303](https://user-images.githubusercontent.com/103327712/233850846-0667766b-a1dd-48b8-b77e-26690bbc8f8b.png)